tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/partners_block.html

```diff
@@ -0,0 +1,12 @@
+{% load wagtailimages_tags %}
+
+{% if value.title %}
+    <h2 class="partners__title heading--two">{{ value.title }}</h2>
+{% endif %}
+
+
+<div class="partners__image-card">
+    {% for logo in value.partner_logos %}
+        {% image logo.image fill-100x100 format-webp alt="" %}
```

Comment:
This needs to be just `{% image logo ... %}` without the `.image` (although that might change if we adapt the block to add optional alt text